base_tier_validation/models/tier_validation.py

```diff
@@ -49,6 +54,17 @@ class TierValidation(models.AbstractModel):
     has_comment = fields.Boolean(compute="_compute_has_comment")
     next_review = fields.Char(compute="_compute_next_review")
 
+    @api.depends("review_ids", "review_ids.status")
+    def _compute_validation_max_sequence(self):
```

Comment:
You probably need to make a migration script for this. Otherwise, it will take a long time to reprocess all existent data.

Reply:
We have fixed in https://github.com/OCA/server-ux/pull/470/commits/fe3cf61f4b634e11675d2e00ac6c0d03c08bf524 making the field non-stored, searchable.